randovania/gui/ui_files/preset_dock_rando.ui

```diff
@@ -89,17 +110,20 @@
              </property>
             </widget>
            </item>
-           <item row="5" column="0" colspan="2">
-            <widget class="QLabel" name="multiworld_label">
+           <item row="9" column="0" colspan="2">
+            <widget class="QLabel" name="two_sided_door_search_label">
              <property name="text">
-              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The selected type is &lt;span style=&quot; font-weight:700;&quot;&gt;not&lt;/span&gt; compatible with multiworld sessions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When using type Doors, enables the use of a revised algorithm for deciding what doors can be.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
i dont like how this can be misinterpreted, but thats already the case for other places too :(